tests/unit/lms/views/api/exceptions_test.py

```diff
@@ -43,23 +44,6 @@ def context(self):
         return CanvasAPIError(explanation="test_explanation", details={"foo": "bar"})
 
 
-class TestLTIOutcomesAPIError:
-    def test_it(self, pyramid_request, views):
-        json_data = views.proxy_api_error()
-
-        assert pyramid_request.response.status_code == 400
-        assert json_data == {
-            "message": "test_explanation",
-            "details": {"foo": "bar"},
-        }
-
-    @pytest.fixture
-    def context(self):
-        return LTIOutcomesAPIError(
-            explanation="test_explanation", details={"foo": "bar"}
-        )
```

Comment:
Deleted this test in a separate commit. The test was misnamed: `TestLTIOutcomesAPIError.test_it()`, but it's actually a test for the `proxy_api_error()` exception view (now `external_request_error()`). The test was also unnecessary: the `TestExternalRequestError` class above has it covered.